rapids-cmake/find/generate_module.cmake

```diff
@@ -103,12 +119,34 @@ Result Variables
   :cmake:variable:`CMAKE_MODULE_PATH` will be modified to include the
   folder where `Find<PackageName>.cmake` is located.
 
+
+Example on how to properly use :cmake:command:`rapids_generate_module`:
+
+.. code-block:: cmake
+
+  ...
+
+  rapids_find_generate_module(
+    RDKAFKA
+    HEADER_NAMES rdkafkacpp.h
+    LIBRARY_NAMES rdkafka++
+    BUILD_EXPORT_SET consumer-exports
+    INSTALL_EXPORT_SET consumer-exports
```

Comment:
Did you want to include an example of the initial and final code blocks? No worries if not, asking given the context of the PR.